lib/Common/ConfigFlagsList.h

```diff
@@ -1477,6 +1487,9 @@ FLAGR(Number, JITServerIdleTimeout, "Idle timeout in seconds to do the cleanup i
 FLAGR(Number, JITServerMaxInactivePageAllocatorCount, "Max inactive page allocators to keep before schedule a cleanup", 10)
 
 FLAGNR(Boolean, StrictWriteBarrierCheck, "Check write barrier setting on none write barrier pages", false)
+FLAGNR(Boolean, WriteBarrierTest, "Always return true while checking barrier to test recycler regardless of annotation", DEFAULT_CONFIG_WriteBarrierTest)
+FLAGNR(Boolean, EnableHardwareWriteBarrier, "Use to turn off write watch to test software write barrier on windows", DEFAULT_CONFIG_EnableHardwareWriteBarrier)
```

Comment:
test the behavior on windows, without rebuilding, after the SWB work is done I think we can remove it.

---
In reply to: [87929333](https://github.com/Microsoft/ChakraCore/pull/1968#discussion_r87929333) [](ancestors = 87929333,87929125)